.github/workflows/ci.yaml

```
@@ -72,7 +72,7 @@ jobs:
         uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf # v4
         with:
           path: ${{env.ucm_local_bin}}
-          key: ucm-${{ matrix.os }}-${{ hashFiles('**/stack.yaml', '**/package.yaml', '**/*.hs', '**/unison-cli-integration/integrationtests/IntegrationTests/*')}}
+          key: ucm-${{ matrix.os }}-${{ hashFiles('**/ci.yaml', '**/stack.yaml', '**/package.yaml', '**/*.hs', '**/unison-cli-integration/integrationtests/IntegrationTests/*')}}
```

```suggestion
          key: ucm-${{ matrix.os }}-${{ hashFiles('**/ci.yaml', '**/stack.yaml', '**/package.yaml', '**/*.hs', '**/unison-cli-integration/integration-tests/IntegrationTests/*')}}
```

See https://github.com/unisonweb/unison/pull/5802#discussion_r2201763403

---

Thanks

---

@sellout do you think it should go here, or in #5802?  I was planning to get this merged first.  But thanks for calling it out regardless.

---

Ok I added it here.

---

Ah right – I saw the addition of ci.yaml here, and I was thinking this was instead of #5802, but there is still other stuff there that should get it.

So yeah, I guess I would leave this whole line (ci.yaml & integration-tests) to #5802, but fine to have it here now, too.